homeassistant/components/bosch_shc/alarm_control_panel.py

```diff
@@ -0,0 +1,137 @@
+"""Platform for alarm control panel integration."""
+from __future__ import annotations
+
+from boschshcpy import SHCIntrusionSystem, SHCSession
+
+from homeassistant.components.alarm_control_panel import AlarmControlPanelEntity
+from homeassistant.components.alarm_control_panel.const import (
+    SUPPORT_ALARM_ARM_AWAY,
+    SUPPORT_ALARM_ARM_CUSTOM_BYPASS,
+    SUPPORT_ALARM_ARM_HOME,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_CUSTOM_BYPASS,
+    STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_ARMING,
+    STATE_ALARM_DISARMED,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from .const import DATA_SESSION, DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the alarm control panel platform."""
+    entities = []
+    session: SHCSession = hass.data[DOMAIN][config_entry.entry_id][DATA_SESSION]
+
+    intrusion_system = session.intrusion_system
+    alarm_control_panel = IntrusionSystemAlarmControlPanel(
+        device=intrusion_system,
+        parent_id=session.information.unique_id,
+        entry_id=config_entry.entry_id,
+    )
+    entities.append(alarm_control_panel)
+
+    async_add_entities(entities)
+
+
+class IntrusionSystemAlarmControlPanel(AlarmControlPanelEntity):
+    """Representation of SHC intrusion detection control."""
+
+    _attr_icon: str = "mdi:security"
```

Comment:
```suggestion
```
Alarm Control Panel entities will have an icon by default. Any specific reason why you want to override this?